core/templates/services/guppy-initialization.service.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for initializing guppy instances.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+class GuppyObject {
+  divId = null;
+  guppyInstance = null;
+  constructor(divId: string, guppyInstance: Guppy) {
+    this.divId = divId;
+    this.guppyInstance = guppyInstance;
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GuppyInitializationService {
+  private guppyInstances: Array<GuppyObject> = [];
+
+  init(guppyDivClassName: string): void {
+    var guppyDivs = document.querySelectorAll('.' + guppyDivClassName);
```

Comment:
Use `const` and `let` instead of `var`. Here and everywhere else (at least in angular2+ code)

Reply:
Done.